enzy_htp/_interface/amber_interface.py

```diff
@@ -324,6 +325,7 @@ def _parameterize_modified_res(self, maa: ModifiedResidue, gaff_type: str) -> Tu
         # init
         fs.safe_mkdir(self.ncaa_param_lib_path)
         target_method = f"{self.charge_method}-{gaff_type}"
+        maa_region = create_region_from_residues(residues=[maa], nterm_cap="H", cterm_cap="OH")
 
         # 0. search parm lib - same as ligand
         mol_desc_path, frcmod_path_list = search_ncaa_parm_file(maa,
```

Comment:
This may not working well for the modified AA. Let's make a unit test in this case. (it will be under test_ncaa_library.py)

Reply:
To be clear, we finish all the following steps that give us frcmod file and do this